src/Features/LanguageServer/Protocol/RoslynRequestExecutionQueue.cs

```diff
@@ -27,5 +32,23 @@ public override Task WrapStartRequestTaskAsync(Task nonMutatingRequestTask, bool
                 return nonMutatingRequestTask.ReportNonFatalErrorAsync();
             }
         }
+
+        private void TrySetLocale()
+        {
+            var locale = _initializeManager.TryGetInitializeParams()?.Locale;
+            // The client may not have given us a UI or this is the initialize request and we haven't saved it yet.
+            if (!string.IsNullOrWhiteSpace(locale))
+            {
+                try
+                {
+                    var desiredUICulture = CultureInfo.CreateSpecificCulture(locale);
+                    CultureInfo.CurrentUICulture = desiredUICulture;
+                }
+                catch (CultureNotFoundException)
+                {
+                    _logger.LogWarning($"Culture {locale} was not found, falling back to OS culture");
```

Comment:
So if the culture doesn't exist, are we going to be logging this warning on every single request?

Reply:
Ah... yeah it will.  Will update this.